examples/features/name_resolving/client/main.go

```diff
@@ -16,7 +16,8 @@
  *
  */
 
-// Binary client is an example client.
+// Binary client demonstrates how to use different name resolvers to connect
```

Comment:
Nit: instead of `connect to a server`, use something like `resolver server backend addresses`.

Also, the core of this example is an implementation of a custom name resolver. That needs to be mentioned in this comment.

Reply:
Done